src/migrations/1751644208200-MergeGroupTables.ts

```
@@ -0,0 +1,85 @@
+    await queryRunner.query(`ALTER TABLE vms DROP COLUMN IF EXISTS group_vm_id`);
+    await queryRunner.query(`ALTER TABLE servers DROP COLUMN IF EXISTS group_server_id`);
+
+    // 11. Drop junction tables
+    await queryRunner.query(`DROP TABLE IF EXISTS group_server_group_vm`);
+
+    // 12. Drop old tables
+    await queryRunner.query(`DROP TABLE IF EXISTS group_vms`);
+    await queryRunner.query(`DROP TABLE IF EXISTS group_servers`);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
```

ESLint

[error] 'queryRunner' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)